src/renderers/noop/ReactNoop.js

```diff
@@ -61,219 +62,231 @@ function flattenChildren(children : HostChildren<Instance | TextInstance>) {
   return flatArray;
 }
 
-var NoopRenderer = ReactFiberReconciler({
```

Comment:
These are just indentation changes + pass `options.debugTool` through to the reconciler.